sdk/storage/azure-storage-blob/azure/storage/blob/container_client.py

```diff
@@ -951,6 +952,262 @@ def delete_blob(
+        if rehydrate_priority is not None:
+            header_parameters['x-ms-rehydrate-priority'] = self._client._serialize.header("rehydrate_priority", rehydrate_priority, 'str')
+        if request_id is not None:
+            header_parameters['x-ms-client-request-id'] = self._client._serialize.header("request_id", request_id, 'str')
+        if lease_id is not None:
+            header_parameters['x-ms-lease-id'] = self._client._serialize.header("lease_id", lease_id, 'str')
+
+        return query_parameters, header_parameters
+
+    @distributed_trace
+    def set_standard_blob_tier_blobs(
+        self,
+        standard_blob_tier,  # type: Union[str, StandardBlobTier]
+        *blobs,  # type: Union[str, BlobProperties]
+        **kwargs
+    ):
+        # type: (...) -> Iterator[HttpResponse]
+        """This operation sets the tier on block blobs.
+
+        A block blob's tier determines Hot/Cool/Archive storage type.
+        This operation does not update the blob's ETag.
+
+        :param blobs: The blobs with which to interact.
+        :type blobs: str or ~azure.storage.blob.models.BlobProperties
+        :param standard_blob_tier:
+            Indicates the tier to be set on the blob. Options include 'Hot', 'Cool',
+            'Archive'. The hot tier is optimized for storing data that is accessed
+            frequently. The cool storage tier is optimized for storing data that
+            is infrequently accessed and stored for at least a month. The archive
+            tier is optimized for storing data that is rarely accessed and stored
+            for at least six months with flexible latency requirements.
+        :type standard_blob_tier: str or ~azure.storage.blob.models.StandardBlobTier
+        :param int timeout:
+            The timeout parameter is expressed in seconds.
+        :param lease:
+            Required if the blob has an active lease. Value can be a LeaseClient object
+            or the lease ID as a string.
+        :type lease: ~azure.storage.blob.lease.LeaseClient or str
+        :rtype: None
+        """
+        access_conditions = get_access_conditions(kwargs.pop('lease', None))
+        if standard_blob_tier is None:
+            raise ValueError("A StandardBlobTier must be specified")
+
+        query_parameters, header_parameters = self._generate_set_tier_options(
+            tier=standard_blob_tier,
+            lease_access_conditions=access_conditions,
+            **kwargs
+        )
+        # To pass kwargs to "_batch_send", we need to remove anything that was
+        # in the Autorest signature for Autorest, otherwise transport will be upset
+        for possible_param in ['timeout', 'lease']:
+            kwargs.pop(possible_param, None)
+
+        reqs = []
+        for blob in blobs:
+            req = HttpRequest(
+                "PUT",
+                "/{}/{}".format(self.container_name, blob),
```

Comment:
The `comp` query parameter isn't included here, but is for the page blob tier below

Reply:
I should just remove it everywhere

Reply:
Removed it in the rest of the places